work.py

```diff
@@ -97,6 +102,28 @@ def spoof_times(self, work_day: dict) -> dict:
 
         return {'start': start_time, 'end': end_time}
 
+    def fixed_times(self, work_day: dict) -> dict:
+        """
+        Generate start and end dates according to the provided parameters.
+        Start time is set to the minimal_start_time value, and
+        end time is set to the maximal_start_time value.
+
+        :param dict work_day: the parsed input from JSON parameters file section [work][work_day]
+        :return: dict with start datetime object and end datetime object representing the start/end of workday
+        """
+        ptn = r'(?P<hour>\d+)\:(?P<minute>\d+)'
+        match_min = re.search(pattern=ptn, string=work_day['minimal_start_time'])
+        match_max = re.search(pattern=ptn, string=work_day['maximal_end_time'])
+        start_time = dt.datetime(
+            year=self._date.year, month=self._date.month, day=self._date.day,
+            hour=int(match_min.group('hour')),
+            minute=int(match_min.group('minute')))
+        end_time = dt.datetime(
+            year=self._date.year, month=self._date.month, day=self._date.day,
+            hour=int(match_max.group('hour')),
+            minute=int(match_max.group('minute')))
```

Comment:
This is a cool use of these parameters.
Use max and min times just with a different mode.

Reply:
Though it can be confusing so better add it to the readme.
Also, I'm not sure what is better using min max or min with normal_day.
Anyway, the chosen method should be documented.

There is some code duplication, so the first few lines could be split into another method for code reuse. your decision.